src/python/grpcio/grpc/__init__.py

```diff
@@ -1506,13 +1559,15 @@ def wait_for_termination(self, timeout=None):
 #################################  Functions    ################################
 
 
-def unary_unary_rpc_method_handler(behavior,
-                                   request_deserializer=None,
-                                   response_serializer=None):
+def unary_unary_rpc_method_handler(
+    behavior: Callable[[Any, Any], Any],
```

Comment:
Not entirely sure here, but based on [example](https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests/qps/benchmark_server.py#L39), all `behavior` should take two arguments, one for request and one for context.

Reply:
You're right. The docstrings are imprecise. We should probably add "and a ServiceContext object" to all 4 of these docstrings.

This reminds me. Can we also look into making sure that [our generated documentation](https://grpc.github.io/grpc/python/grpc.html) displays these new types?